### ISSUE TITLE
Please add support for latest th-orphans and syb

### ISSUE THREAD

**Issue by peti** [bmillwood/haskell-src-meta#43]
http://packdeps.haskellers.com/feed?needle=haskell-src-meta

Currently, the build fails like so:

~~~~~~~~~~~
src/Language/Haskell/Meta/Utils.hs:67:1:
    Duplicate instance declarations:
      instance Typeable Q
        -- Defined at src/Language/Haskell/Meta/Utils.hs:67:1
      instance Typeable Q -- Defined in ‘Language.Haskell.TH.Instances’

src/Language/Haskell/Meta/Utils.hs:71:1:
    Duplicate instance declarations:
      instance Typeable QuasiQuoter
        -- Defined at src/Language/Haskell/Meta/Utils.hs:71:1
      instance Typeable QuasiQuoter
        -- Defined in ‘Language.Haskell.TH.Instances’
~~~~~~~~~~~~~

**Comment by bmillwood** [comment]
Can't reproduce this. What versions are you using?

**Comment by peti** [comment]
It's probably not so important to try and re-produce exactly the error I ran into while running a patched build. What matters is that there is an official release version of `haskell-src-meta` that accepts current versions of `th-orphans` and `syb`.

**Comment by ddssff** [comment]
The Typeable instances appeared in th-orphans-0.12.0

**Comment by peti** [comment]
Ping?

**Comment by snoyberg** [comment]
This is also a problem for Stackage: https://github.com/fpco/stackage/issues/591

**Comment by mainland** [comment]
Fixed in version 0.6.0.10.
